Automattic/wp-calypso
# Stats: fix some details on the Posting Activity section

#### Proposed Changes

* Fix the horizontal margins on mobile
* Refactor the CSS by moving the tooltip CSS rules to a more appropriate CSS file

#### Testing Instructions

* Go to the Calypso live link
* Navigate to the Insights tab
* Ensure the Posting Activity section has a horizontal margin on mobile
* Ensure the tooltip on the Posting Activity looks the same as before (see the screenshot below)

#### Screenshots

![image](https://user-images.githubusercontent.com/6406071/210602672-eb5ee87f-e66f-48a5-9076-ee2faa176567.png)
![image](https://user-images.githubusercontent.com/6406071/210603087-1b99c792-7c44-474b-ae90-fb74212d9f99.png)




#### Pre-merge Checklist





Related to #70609
